### PR TITLE
Verify update

### PR DESCRIPTION
**Pull request by dperpeet**

### CODE REVIEW

File: verify/README [hunk 1]
```diff
@@ -31,7 +39,17 @@ update github status:
     /home/cockpit/.ssh/id_rsa
     /home/cockpit/.ssh/id_rsa.pub
     /home/cockpit/.config/github-token
-    /home/cockpit/.config/github-whitelist
```
Comment (mvollmer) [inline verify/README]: Why is this removed?

Comment (dperpeet) [inline verify/README]: Right, I thought we didn't use it anymore. I'll change the docs.
